contracts/contracts/solidity/library/NamedStorage.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: Apache-2.0.
```

Comment:
Aha this is a vendor contract: https://github.com/starkware-libs/starkgate-contracts/blob/c08863a1f08226c09f1d0748124192e848d73db9/src/starkware/starknet/solidity/StarknetMessaging.sol

We should really try to import and use them from the source.

Ideally we have them packaged and distributed, but looks this code is not packaged via PyPI or NPM ATM.

Let's try a submodule under the `./vendor` folder. Here is an example like we have it setup on the LinkToken project: https://github.com/smartcontractkit/LinkToken/tree/master/vendor